Dockerfile_ssportal

```diff
@@ -0,0 +1,23 @@
+FROM node:16-alpine
```

Comment:
Just a question, I thought we are upgrading to node 18, why not use the node 18 image in the image?

Reply:
Good call, updated